### PR TITLE
Resources cleanup

### PR DESCRIPTION
**Pull request by michaelbukachi**
## What is the Purpose?
Setup logic for UI tests and remove duplicate resources

## What was the approach?
General cleanup and addition of test dependencies

## Are there any concerns to addressed further before or after merging this PR?
UI tests are still not working
Styling might be broken for some screens

## Mentions?
@wangerekaharun @chepsi @JabezNzomo99 

## Issue(s) affected?
#78

### CODE REVIEW

File: features/feed/src/main/java/com/android254/droidconKE2020/feed/ui/views/ShareFragment.kt [hunk 1]
```diff
@@ -7,13 +7,10 @@ import android.view.ViewGroup
 import android.widget.Toast
```
Comment (wangerekaharun) [inline features/feed/src/main/java/com/android254/droidconKE2020/feed/ui/views/ShareFragment.kt:7]: @michaelbukachi  Why the shift from View Binding to Kotlin Synthetics?

Comment (michaelbukachi) [reply features/feed/src/main/java/com/android254/droidconKE2020/feed/ui/views/ShareFragment.kt:7]: I'll change it back. I'm trying to figure out what's wrong with UI tests. Doing process of elimination.

Comment (wangerekaharun) [reply features/feed/src/main/java/com/android254/droidconKE2020/feed/ui/views/ShareFragment.kt:7]: Okay no worries then

Comment (JabezNzomo99) [reply features/feed/src/main/java/com/android254/droidconKE2020/feed/ui/views/ShareFragment.kt:7]: What's the progress on this, i want to make a PR on issue #54 and i want to pull the resources clean up changes

Comment (michaelbukachi) [reply features/feed/src/main/java/com/android254/droidconKE2020/feed/ui/views/ShareFragment.kt:7]: Let me merge into develop. I'll still work on this branch though to work on the Ci bit

**Pull request merged**